frontend/app/src/components/display/AmountDisplay.vue

```diff
@@ -166,6 +128,52 @@ export default class AmountDisplay extends Vue {
     }
     return valueToRender;
   }
+
+  get convertFiat(): boolean {
+    const { ticker_symbol } = this.currency;
+    return !!this.fiatCurrency && this.fiatCurrency !== ticker_symbol;
+  }
+
+  get formattedValue(): string {
+    return this.formatPrice(this.renderValue);
+  }
+
+  get fullValue(): string {
+    return this.convertFiat
+      ? this.convertPrice(this.renderValue).toFormat(2)
+      : this.renderValue.toFormat(this.renderValue.decimalPlaces());
+  }
+
+  private convertPrice(value: BigNumber): BigNumber {
```

Comment:
Sure I just moved that from the filters, but it should be easy to rename. Also a note, we should probably start moving away from filters since they are gonna be removed from vue 3.

Reply:
Really? That sucks :(

Reply:
Yes https://v3.vuejs.org/guide/migration/filters.html#overview